cosmic/sample/sampler/independent.py

```diff
@@ -408,12 +409,12 @@ def sample_porb(self, mass1, mass2, ecc, model='sana12', size=None):
             porb = porb_yr*yr_day
         elif model == 'sana12':
             from cosmic.utils import rndm
-            porb = 10**rndm(a=0.15, b=5.5, g=0.55, size=size)
+            porb = 10**rndm(a=0.15, b=5.5, g=0.45, size=size)
```

Comment:
Are you sure about this? From the supplemental materials of [Sana+2012](https://science.sciencemag.org/content/sci/suppl/2012/07/25/337.6093.444.DC1/1223344.Sana.SM.pdf), it indeed looks like the exponent for the orbital period power law (\pi) is -0.55

Reply:
Oh I see...If `g` is indeed 1+x where x is the powerlaw exponent, then we're gucci. 

But to be honest, for the sake of clarity, why don't we just feed in the powerlaw exponent directly into rndm and add 1 inside of that function?

Reply:
yeah. I'll fix the rndm method in utils. I agree that this makes the most sense since future users won't have to make the same 'omg what is a power law' journey I made yesterday.